src/mercury-common/test-requirements.txt

```diff
@@ -1,3 +1,5 @@
 mock
 pytest
 pytest-cov
+pytest-asyncio
+pytest-async-mongodb
```

Comment:
mercury-common is a requirement of mercury-rpc, but this test-requirements.txt should probably live with the mercury-rpc package.